### PR TITLE
add dismissable announcement system

### PR DESCRIPTION
**Pull request by dsifford**
cc: @jvoros

There's quite a bit going on here so I'll spare you the nitty gritty details.

In a nutshell, this PR allows users to dismiss announcements by saving a token in localstorage. The announcements are served from a new `announcements.yml` file in the cards directory.

Let me know what you think!

### CODE REVIEW

File: app/components/Announcements.tsx [hunk 1]
```diff
@@ -1,11 +1,76 @@
...
+      heading
+      message
+    }
+  }
+`;
+
+@graphql(announcements)
+export default class Announcements extends React.Component<Props, State> {
+
+  static lastSeen: string|null = localStorage.getItem(LOCALSTORAGE_KEY);
```
Comment (jvoros) [inline app/components/Announcements.tsx]: You and the statics ;)

File: app/components/Announcements.tsx [hunk 2]
```diff
@@ -1,11 +1,76 @@
...
+    }
+    this.setState(prevState => ({ ...prevState, isDismissed: true }));
+  }
+
+  render() {
+    if (!this.state.announcement || this.state.isDismissed) { return null; }
+    return (
+      <div className="announcement">
+          <div className="announcement_content">
+            <span className="foo">
```
Comment (jvoros) [inline app/components/Announcements.tsx]: foo?

Comment (dsifford) [inline app/components/Announcements.tsx]: Haha whoops... That was me testing something 😊

File: app/components/Announcements.tsx (hunk 1 above)
Comment (dsifford) [inline app/components/Announcements.tsx]: Haha it felt wrong putting that in the state since it never changes directly

**Pull request merged**